pom.xml

```diff
@@ -43,7 +43,7 @@
 
     <!-- Versions of the API model and tools: -->
     <metamodel.version>1.3.7</metamodel.version>
-    <model.version>4.5.10</model.version>
+    <model.version>4.5.11-SNAPSHOT</model.version>
```

Comment:
I don't think we ever published a SNAPSHOT version and so we bump the version only once the new model version is published

Reply:
Right. This is only for local testing. Should I change it to `4.5.11` ?

Reply:
No, it should remain 4.5.10 and @oliel  will bump the version after releasing 4.5.11